config/settings.d/puppet_proxy_legacy.yml.example

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Could you add some comments to the top of this and the api config file to indicate when they're used?  Perhaps also to puppet.yml near puppet_version so people know they exist.  It may be unclear to users which is in use.